ocs_ci/ocs/bucket_utils.py

```diff
@@ -158,6 +158,45 @@ def retrieve_anon_s3_resource():
     return anon_s3_resource
 
 
+def list_objects_from_bucket(
+    podobj, target, prefix=None, s3_obj=None, signed_request_creds=None, **kwargs
+):
+    """
+    Lists objects in a bucket using s3 ls command
+
+    Args:
+        podobj: Pod object that is used to perform copy operation
+        src_obj: full path to object
+        target: target bucket
+        prefix: prefix
+        s3_obj: obc/mcg object
+    Returns:
+        List of objects in a bucket
+    """
+    if prefix:
+        retrieve_cmd = f"ls {target}/{prefix}"
+    else:
+        retrieve_cmd = f"ls {target}"
```

Comment:
Shouldn't the object listed with --recursive option?

Reply:
nope. when you list with --recursive it works fine..and gives the count of output